integration_tests/models/schema_tests/schema.yml

```diff
@@ -117,6 +117,9 @@ models:
             max_value: 4
         - dbt_expectations.expect_table_row_count_to_equal_other_table:
             compare_model: ref("data_test")
+        - dbt_expectations.expect_table_row_count_to_equal_other_table_times_factor:
+            compare_model: ref("data_test")
+            factor: 1
```

Comment:
Should we maybe introduce a new test column that is factored up something other than `1`?